smoketest/scripts/cli/test_interfaces_bridge.py

```diff
@@ -51,6 +51,7 @@ def setUp(self):
             self.session.set(['interfaces','bonding','bond1','member','interface','dum0'])
             self.session.set(['interfaces','bonding','bond1','member','interface','dum1'])
             self._members.append('bond1')
+            self._mirror_interfaces.append('bond1')
```

Comment:
This should not be done for ALL testscases, rather limit this to the specific test.